### PR TITLE
matrix creator instantiation

### PR DESCRIPTION
**Pull request by nicolacavallini**
Codimension one instantiation of create_laplace_matrix

### CODE REVIEW

File: source/numerics/matrix_creator.inst.in [hunk 1]
```diff
@@ -305,80 +305,42 @@ for (scalar : COMPLEX_SCALARS; deal_II_dimension : DIMENSIONS; deal_II_space_dim
...
+       const Function<deal_II_space_dimension,double>      &rhs,
        Vector<double>           &rhs_vector,
-       const Function<deal_II_dimension> * const coefficient,
+       const Function<deal_II_space_dimension,double> * const coefficient,
        const ConstraintMatrix   &constraints);
-
-// hp versions of create_laplace_matrix
-    template
-      void MatrixCreator::create_laplace_matrix<deal_II_dimension>
-      (const hp::DoFHandler<deal_II_dimension>    &dof,
```
Comment (bangerth) [inline source/numerics/matrix_creator.inst.in]: So what happened to these function instantiations?

Comment (nicolacavallini) [inline source/numerics/matrix_creator.inst.in]: Compiler said I had a repetition and I was lazy checking :)

I'll fix this :)

Comment (nicolacavallini) [inline source/numerics/matrix_creator.inst.in]: Yep clang is telling me that:
'create_laplace_matrix<1, 1>'

counts as:
create_laplace_matrix< 1 >

today i'll check if gcc agrees...

Comment (luca-heltai) [inline source/numerics/matrix_creator.inst.in]: It's not a matter of <1> or <1,1>. You just removed all instantiations for `hp::DoFHandler`...

File: source/numerics/matrix_creator.inst.in [hunk 2]
```diff
@@ -303,6 +303,7 @@ for (scalar : COMPLEX_SCALARS; deal_II_dimension : DIMENSIONS; deal_II_space_dim
 #endif
   }
 
+
```
Comment (davydden) [inline source/numerics/matrix_creator.inst.in]: i guess you don't need this change ;-)

Comment (nicolacavallini) [inline source/numerics/matrix_creator.inst.in]: I hate blanked lines, sorry about that :)

**Pull request merged**